src/status_im/chat/models.cljs

```diff
@@ -248,40 +248,33 @@
 
 (fx/defn navigate-to-chat
   "Takes coeffects map and chat-id, returns effects necessary for navigation and preloading data"
-  [cofx chat-id {:keys [modal? navigation-reset?]}]
-  (cond
-    modal?
-    (fx/merge cofx
-              (navigation/navigate-to-cofx :chat-modal {})
-              (preload-chat-data chat-id))
-
-    :else
-    (fx/merge cofx
-              (navigation/navigate-to-cofx :chat {})
-              (preload-chat-data chat-id))))
+  [cofx chat-id]
+  (fx/merge cofx
+            (navigation/navigate-to-cofx :chat {})
+            (preload-chat-data chat-id)))
 
 (fx/defn start-chat
   "Start a chat, making sure it exists"
-  [{:keys [db] :as cofx} chat-id opts]
+  [{:keys [db] :as cofx} chat-id _]
```

Comment:
What's the reason to keep parameter's placeholder here?

Reply:
further refactoring needed to remove it from other methods